backend/btrixcloud/operator/crawls.py

```diff
@@ -274,12 +274,9 @@ async def sync_crawls(self, data: MCSyncData):
         params["storage_path"] = storage_path
         params["storage_secret"] = storage_secret
 
-        # only resolve if not already set
-        # not automatically updating image for existing crawls
-        if not status.crawlerImage:
-            status.crawlerImage = self.crawl_config_ops.get_channel_crawler_image(
-                crawl.crawler_channel
-            )
+        status.crawlerImage = self.crawl_config_ops.get_channel_crawler_image(
+            crawl.crawler_channel
+        )
```

Comment:
Just confirming we want to update the image for crawls that are already running, since this is a change in behavior. Wasn't sure if it was held over from the larger PR.

Reply:
Ah, yeah it was from larger PR, but sort of how it was intended to work, this check made it harder to update the image to latest in channel, which is generally what we want - makes it easier to fix issue with a crawl that require updating to latest image.